### ISSUE TITLE
Support feeder

### ISSUE THREAD

**Issue by 3tty0n** [3tty0n/gatling-thrift#5]
http://gatling.io/docs/current/session/feeder/

**Comment by 3tty0n** [comment]
use case:
- http://gatling.io/docs/current/advanced_tutorial/#step-03-use-dynamic-data-with-feeders-and-checks

**Comment by komsit37** [comment]
maybe something like this scenario

```scala
  val orderFeed = csv("orders.csv").random
  /* orders.csv
  accountId, sym, qty, price
  1L, 7203.T, 10, 6700
  2L, 7100.T, 5, 322
  3L, 8100.T, 40, 788
   */
  scenario("submit order")
    .thriftClient[Dealing[Future]]("localhost:9001")
    .feed(orderFeed)
    .thriftRequest({
      (client, (accountId, sym, qty, price)) =>
      client.submitOrder(accountId, OrderLine(Side.Buy, sym, qty, OrderType.Limit, Some(price)))
    })
```

**Comment by 3tty0n** [comment]
This can be useful when implementing feeder: https://github.com/gatling/gatling/blob/master/gatling-jdbc/src/main/scala/io/gatling/jdbc/feeder/JdbcFeederSource.scala

**Comment by 3tty0n** [comment]
It will be resolved by #10

**Comment by 3tty0n** [comment]
It is resolved by https://github.com/3tty0n/gatling-thrift/commit/96108f17f646db15a79675e6eeea1bcb573738c2 .
